docs/root/start/sandboxes/_include/envoy-opentelemetry.yaml

```diff
@@ -0,0 +1,120 @@
+static_resources:
```

Comment:
we probably dont need to add this - we can use the file here (eg) https://github.com/envoyproxy/envoy/blob/main/examples/opentelemetry/envoy-front-proxy.yaml with `_include/opentelemetry/envoy-front-proxy.yaml`

Reply:
Done in d01a66c186